app/serializers/rest/tag_serializer.rb

```diff
@@ -3,10 +3,12 @@
 class REST::TagSerializer < ActiveModel::Serializer
   include RoutingHelper
 
-  attributes :name, :url, :history
+  attributes :id, :name, :url, :history
```

Comment:
I'm not sure if the `id` of the Tag that's internal to the servers' database was originally not exposed for a certain reason or not (e.g., security considerations)

Another way we could achieve this would be to change the routing on `/admin/tags/...` to accept the `:name` instead of the `:id` (and if the `:name` is only numbers, then redirect to the canonical form using the hashtag's name). I'm not sure if that could expose attacks similar to lookalike characters or other "weirdness" though.